### ISSUE TITLE
Connection leak detection triggered for conn18

### ISSUE THREAD

**Issue by flamechair** [Minecolonies/Rankup#19]
```
[15:06:40] [HikariPool-2 housekeeper/WARN] [com.zaxxer.hikari.pool.ProxyLeakTask]: Connection leak detection triggered for conn18: url=jdbc:h2:/home/container/./config/rankup/h2/playerstats user=, stack trace follows
java.lang.Exception: Apparent connection leak detected
at com.minecolonies.rankup.util.AccountingUtils.getConn(AccountingUtils.java:96) ~[AccountingUtils.class:?]
at com.minecolonies.rankup.util.AccountingUtils.createTableIfNeeded(AccountingUtils.java:147) ~[AccountingUtils.class:?]
at com.minecolonies.rankup.modules.databases.DatabaseModule.Rankup2Enable(DatabaseModule.java:22) ~[DatabaseModule.class:?]
at com.minecolonies.rankup.qsml.modulespec.StandardModule.onEnable(StandardModule.java:92) ~[StandardModule.class:?]
at uk.co.drnaylor.quickstart.loaders.ModuleEnabler.enableModule(ModuleEnabler.java:51) ~[ModuleEnabler.class:?]
at uk.co.drnaylor.quickstart.ModuleContainer$EnablePhase$2.onModuleAction(ModuleContainer.java:856) ~[ModuleContainer$EnablePhase$2.class:?]
at uk.co.drnaylor.quickstart.ModuleContainer.loadModules(ModuleContainer.java:476) ~[ModuleContainer.class:?]
at com.minecolonies.rankup.Rankup.onInit(Rankup.java:135) ~[Rankup.class:?]
at org.spongepowered.common.event.listener.GameInitializationEventListener_Rankup_onInit5.handle(Unknown Source) ~[?:?]
at org.spongepowered.common.event.RegisteredListener.handle(RegisteredListener.java:95) ~[spongeforge-1.12.2-2705-7.1.0-BETA-3358.jar:1.12.2-2705-7.1.0-BETA-3358]
at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:383) ~[spongeforge-1.12.2-2705-7.1.0-BETA-3358.jar:1.12.2-2705-7.1.0-BETA-3358]
at org.spongepowered.mod.event.SpongeModEventManager.extendedPost(SpongeModEventManager.java:443) ~[spongeforge-1.12.2-2705-7.1.0-BETA-3358.jar:1.12.2-2705-7.1.0-BETA-3358]
at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:404) ~[spongeforge-1.12.2-2705-7.1.0-BETA-3358.jar:1.12.2-2705-7.1.0-BETA-3358]
at org.spongepowered.common.SpongeImpl.postEvent(SpongeImpl.java:231) ~[spongeforge-1.12.2-2705-7.1.0-BETA-3358.jar:1.12.2-2705-7.1.0-BETA-3358]
at org.spongepowered.mod.SpongeMod.onStateEvent(SpongeMod.java:271) ~[spongeforge-1.12.2-2705-7.1.0-BETA-3358.jar:1.12.2-2705-7.1.0-BETA-3358]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_171]
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_171]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_171]
at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_171]
at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91) ~[minecraft_server.1.12.2.jar:?]
at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150) ~[minecraft_server.1.12.2.jar:?]
at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76) ~[minecraft_server.1.12.2.jar:?]
at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399) ~[minecraft_server.1.12.2.jar:?]
at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71) ~[minecraft_server.1.12.2.jar:?]
at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116) ~[minecraft_server.1.12.2.jar:?]
at com.google.common.eventbus.EventBus.post(EventBus.java:217) ~[minecraft_server.1.12.2.jar:?]
at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:218) ~[forge-1.12.2-14.23.4.2705-universal.jar:?]
at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:196) ~[forge-1.12.2-14.23.4.2705-universal.jar:?]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_171]
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_171]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_171]
at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_171]
at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91) ~[minecraft_server.1.12.2.jar:?]
at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150) ~[minecraft_server.1.12.2.jar:?]
at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76) ~[minecraft_server.1.12.2.jar:?]
at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399) ~[minecraft_server.1.12.2.jar:?]
at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71) ~[minecraft_server.1.12.2.jar:?]
at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116) ~[minecraft_server.1.12.2.jar:?]
at com.google.common.eventbus.EventBus.post(EventBus.java:217) ~[minecraft_server.1.12.2.jar:?]
at net.minecraftforge.fml.common.LoadController.redirect$onPost$zza000(LoadController.java:560) ~[LoadController.class:?]
at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:135) ~[LoadController.class:?]
at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:744) ~[Loader.class:?]
at net.minecraftforge.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:107) ~[FMLServerHandler.class:?]
at net.minecraftforge.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:337) ~[FMLCommonHandler.class:?]
at net.minecraft.server.dedicated.DedicatedServer.init(DedicatedServer.java:219) ~[nz.class:?]
at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486) ~[MinecraftServer.class:?]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_171]

```

**Comment by TheFlash787** [comment]
This is also confirmed on our servers too.

**Comment by flamechair** [comment]
Fixed in #20
